base/applications/drwtsn32/main.cpp

```diff
@@ -177,8 +177,7 @@ std::wstring Settings_GetOutputPath(void)
     {
         UseDefaultPath = TRUE;
     }
-
-    if (key.QueryStringValue(L"Dump Directory", Buffer, &BufferSize) != ERROR_SUCCESS)
+    else if (key.QueryStringValue(L"Dump Directory", Buffer, &BufferSize) != ERROR_SUCCESS)
```

Comment:
Can't we simplify all that with:
```c
    if ((key.Open(HKEY_CURRENT_USER, L"SOFTWARE\\ReactOS\\Crash Reporter", KEY_READ) != ERROR_SUCCESS) ||
        (key.QueryStringValue(L"Dump Directory", Buffer, &BufferSize) != ERROR_SUCCESS))
    {
        UseDefaultPath = TRUE;
    }
```

Reply:
Well, I did not do it, to avoid usual complains about touching more lines than required!
Now, you ask for it... Please, get explicit approval from @learn-more.